site/docs/be/docker.md

```diff
@@ -0,0 +1,4 @@
+---
+layout: redirect
+redirect: docs/be/docker.html
```

Comment:
This `redirect` code is redirecting to https://bazel.build/versions/master/docs/be/docker.html
See its source at https://github.com/bazelbuild/bazel/blob/master/site/_layouts/redirect.html